.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     steps:
       - name: terraform-cloud-outputs
         id: tflocal
-        uses: hashicorp-forge/terraform-cloud-action/outputs@v1
+        uses: hashicorp-forge/terraform-cloud-action/outputs@4adbe7eea886138ac10a4c09e63c5c568aaa6672
```

Comment:
It worked!

![Screenshot 2023-05-12 at 10 36 06 AM](https://github.com/hashicorp/go-tfe/assets/174332/7303ca12-bf8a-459b-bc57-d36446b0c5c8)

Nit: can you pin all the terraform-cloud-action/ actions to 4adbe7eea886138ac10a4c09e63c5c568aaa6672 ?

Reply:
I noticed that not all of the tokens were listed so I fixed the reference to use hyphens for real (in my mind I always translate - to _)